ast/src/builtin.rs

```diff
@@ -127,6 +127,21 @@ pub enum Constant {
     Ellipsis,
 }
 
+impl Constant {
+    pub fn is_true(self) -> bool {
+        self.bool().map_or(false, |b| b)
+    }
+    pub fn is_false(self) -> bool {
+        self.bool().map_or(false, |b| !b)
+    }
+    pub fn complex(self) -> Option<(f64, f64)> {
```

Comment:
Nit: I think `as_complex()` or `into_complex()` would be more explicit because it indicates that a conversion is happening. Using `complex()` on its own sounds to me as if it is a simple getter.

Reply:
Because other accessors are synthesized by `is-macro`, I'd like to its convention for other similar methods too.

Here is a short discussion why it is not `into_` but just plain name.

https://github.com/kdy1/is-macro/issues/12

`as_` are named as the convention as you expected.

Reply:
I am ok with either names. But I'd like to avoid putting more effort only to rename them. Let's keep it if it is acceptable (though it seems a bit uncomfortable)

Reply:
Most of the pattern matching in Ruff seems using reference. I can easily find places to use `as_` but not about conversions. It will not be that hard to even rename it later.